tests/post/steps/test_service_configuration.py

```diff
@@ -180,6 +186,47 @@ def restore_csc(
     )
 
 
+@then(parsers.parse(
+    "we have an alert rule '{rule_name}' in group '{group_name}' with "
+    "'{path}' equal to '{value}'"
+))
```

Comment:
Would make sense to define this new parser in the same commit that uses it.

Reply:
Totally.